qa/workunits/rbd/concurrent.sh

```diff
@@ -88,9 +88,15 @@ function setup() {
 
 	# Set up some environment for normal teuthology test setup.
 	# This really should not be necessary but I found it was.
-	TOP="/tmp/cephtest"
+
+        if [ -f /etc/ceph/ceph.conf ]; then
+            TOP="/etc/ceph"
+            export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/ceph.client.0.keyring"
+        else
+            TOP="/tmp/cephtest"
+            export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/data/client.0.keyring"
+        fi
 	export CEPH_ARGS="--conf ${TOP}/ceph.conf"
-	export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/data/client.0.keyring"
```

Comment:
@idryomov it will not overwrite within the script, but if the value is passed from outside the first line will initialize with new value from "/tmp/cephtest" no matter what value is passed from teuthology, I will test only with name and check if it works.